src/fragments/lib/graphqlapi/native_common/advanced-workflows/common.mdx

```diff
@@ -175,3 +175,31 @@ import android9 from '/src/fragments/lib/graphqlapi/android/advanced-workflows/5
 import flutter8 from '/src/fragments/lib/graphqlapi/flutter/advanced-workflows/50_interceptor.mdx';
 
 <Fragments fragments={{ flutter: flutter8 }} />
+
+<InlineFilter filters={['swift']}>
+
+## Custom Request Inputs as JSON Objects
+
+If you have custom query inputs defined, you can convert the input objects to Json objects as part of the requests.
+
+```swift
+extension GraphQLRequest {
+    static func get(filter: CustomRequestFilter) -> GraphQLRequest<JSONValue> {
```

Comment:
I think some of this is ambiguous. We should start off with a schema example that represents the use case, "custom query inputs". Then define the CustomRequestFilter type, looking at the code below, it's at least Encodable. When the developer brings the code over to their IDE it should compile and run

Reply:
I've updated the code sample based on your feedback